staging/src/k8s.io/apiserver/pkg/admission/configuration/configuration_manager.go

```diff
@@ -32,10 +32,7 @@ const (
 	defaultBootstrapGraceperiod = 5 * time.Second
 )
 
-var (
-	ErrNotReady = fmt.Errorf("configuration is not ready")
-	ErrDisabled = fmt.Errorf("disabled")
-)
```

Comment:
"k8s.io/apiserver" is a library, so it could be used outside of k8s. I don't think we can safely remove public symbols now.

Reply:
@caesarxuchao make sense. 
/lgtm cancel